pep-0008.txt

```diff
@@ -465,12 +465,12 @@ Avoid extraneous whitespace in the following situations:
 - Immediately before a comma, semicolon, or colon::
 
       # Correct:
-      if x == 4: print x, y; x, y = y, x
+      if x == 4: print(x, y); x, y = y, x
 
   ::
 
       # Wrong:
-      if x == 4 : print x , y ; x , y = y , x
+      if x == 4 : print( x , y ); x , y = y , x
```

Comment:
```suggestion
      if x == 4 : print(x , y) ; x , y = y , x
```

The purpose of this example was to demonstrate that a space should not be used "immediately before a comma, semicolon, or a colon". The change above removes the space from before the semicolon in the wrong example, and adds it both before and after a parenthesis, which is not discussed here. Therefore, I've revised this to move the space back where it belongs, while still preserving your syntax fix.